reporting-analysis/upd_wsdl_location.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
all wsdl referenze ( wsdl,xsd files and pom too ) should be update

Reply:
to remove, it is not needed